### ISSUE TITLE
Cannot find module 'relay-compiler/lib/RelayCompilerMain'

### ISSUE THREAD

**Issue by pie6k** [relay-tools/vscode-apollo-relay#17]
I'm trying to set up this plugin and I'm getting such error in vs console:

```
[ERROR] A config file failed to load with options: {"configPath":"/Users/user/app","requireConfig":true}.
    The error was: Error: Cannot find module 'relay-compiler/lib/RelayCompilerMain'
    at Object.loadConfig (/Users/user/.vscode/extensions/apollographql.vscode-apollo-1.10.0/node_modules/apollo-language-server/lib/config/loadConfig.js:38:34)
	at client.onReady.then.client.onNotification.message (/Users/user/.vscode/extensions/apollographql.vscode-apollo-1.10.0/lib/extension.js:44:35)
	at handleNotification (/Users/user/.vscode/extensions/apollographql.vscode-apollo-1.10.0/node_modules/vscode-jsonrpc/lib/main.js:489:43)
	at processMessageQueue (/Users/user/.vscode/extensions/apollographql.vscode-apollo-1.10.0/node_modules/vscode-jsonrpc/lib/main.js:260:17)
	at Immediate.setImmediate [as _onImmediate] (/Users/user/.vscode/extensions/apollographql.vscode-apollo-1.10.0/node_modules/vscode-jsonrpc/lib/main.js:247:13)
```

My `apollo.config.js` is 

```ts
const { config } = require('vscode-apollo-relay').generateConfig();
module.exports = config;
```

Relay compiler version is `6.0.0`

Indeed, when I check content of `react-compiler` package: `node_modules/relay-compiler/lib` by running `cd` I get 

```
bin         core        index.js    reporters   util
codegen     handlers    language    transforms  validations
```

**Comment by alloy** [comment]
Ah yes, relay v6 was released yesterday and changed layout slightly. I'll try to get to this asap.

**Comment by alloy** [comment]
This was fixed in v1.4.1 🚀
